src/components/Sum/Sum.jsx

```diff
@@ -1 +1,10 @@
-// export const Sum = () => ();
+export const Sum = ({ a, b }) => (
+  <p>
+    {`Sum of ${a} and ${b} is ${a + b}`}
+  </p>
+);
+
+Sum.defaultProps = {
+  a: 0,
+  b: 0,
+};
```

Comment:
Also you can set default value this way
```suggestion
export const Sum = ({ a= 0, b = 0 }) => (
  <p>
    {`Sum of ${a} and ${b} is ${a + b}`}
  </p>
);
```